code_schemes/s01e01.json

```diff
@@ -40,7 +40,7 @@
         "DisplayText": "meta: answer",
         "NumericValue": -1000140,
         "StringValue": "answer",
-        "VisibleInCoda": true,
+        "VisibleInCoda": false,
```

Comment:
Does this need a full deprecation or is it fine just to hide in Coda?

Reply:
fine to hide in coda